data/secscan_model/__init__.py

```diff
@@ -17,63 +18,53 @@
 
 class SecurityScannerModelProxy(SecurityScannerInterface):
     def configure(self, app, instance_keys, storage):
-        # TODO(alecmerdler): Just use `V4SecurityScanner` once Clair V2 is removed.
         try:
-            self._model = V2SecurityScanner(app, instance_keys, storage)
+            self._model = V4SecurityScanner(app, instance_keys, storage)
         except InvalidConfigurationException:
-            self._model = NoopV2SecurityScanner()
+            self._model = NoopV4SecurityScanner()
 
         try:
-            self._v4_model = V4SecurityScanner(app, instance_keys, storage)
+            self._legacy_model = V2SecurityScanner(app, instance_keys, storage)
         except InvalidConfigurationException:
-            self._v4_model = NoopV4SecurityScanner()
-
-        self._v4_namespace_whitelist = app.config.get("SECURITY_SCANNER_V4_NAMESPACE_WHITELIST", [])
+            self._legacy_model = NoopV2SecurityScanner()
 
         logger.info("===============================")
-        logger.info("Using split secscan model: `%s`", [self._model, self._v4_model])
-        logger.info("v4 whitelist `%s`", self._v4_namespace_whitelist)
+        logger.info("Using split secscan model: `%s`", [self._legacy_model, self._model])
         logger.info("===============================")
 
         return self
 
     def perform_indexing(self, next_token=None):
-        if next_token is None:
-            return SplitScanToken("v4", self._v4_model.perform_indexing(None))
-
-        if next_token.version == "v4" and next_token.token is not None:
-            return SplitScanToken("v4", self._v4_model.perform_indexing(next_token.token))
+        if next_token is None or next_token.version != "v4":
```

Comment:
Can't we just send this to V4 directly now?

Reply:
If we want to get rid of the `SplitToken` completely now, then yes.

Reply:
Yes please. I don't think we need it anymore.